oci-unit-tests/helper/test_helper.sh

```diff
@@ -129,14 +129,20 @@ install_container_packages()
 }
 
 # $1: container id
+# ${2...}: required manifest files
 check_manifest_exists()
 {
     local id="${1}"
+    shift
     local manifest_dir="/usr/share/rocks"
-    local found=0
```

Comment:
Can you instead initialize `$found` with an empty value instead?  This way, we make sure it stays local when it's assigned down below.

Reply:
Thanks, Sergio.

Sure! I just pushed the change [1].

[1] https://github.com/canonical/server-test-scripts/compare/b48487295c744c9394112dcb524eebb7bfb2c1f2..caf165ebe436d6ac02eb723cdb343c02a1550842

Reply:
Awesome, thanks.